src/api/models/iuf/shared.go

```diff
@@ -48,6 +48,21 @@ type InputParameters struct {
 	Force                                 bool                    `json:"force"`                                         // Force re-execution of stage operations
 } //	@name	InputParameters
 
+type InputParametersPatch struct {
+	MediaDir                              *string                  `json:"media_dir"`                                     // Location of media
+	SiteParameters                        *string                  `json:"site_parameters"`                               // DEPRECATED: use site_parameters at the top level of the activity or session resource. The inline contents of the site_parameters.yaml file.
+	LimitManagementNodes                  *[]string                `json:"limit_management_nodes"`                        // Each item is the xname of a management node
+	LimitManagedNodes                     *[]string                `json:"limit_managed_nodes"`                           // Each item is the xname of a managed node
```

Comment:
I think the comments on both of these are wrong. From the Jira, I can see that `limit_management_nodes` is usually of the form `<role>_<subrole>`, and I think that `limit_managed_nodes` can be anything accepted by BOS as the value to a session's `limit` parameter.

I know this is just copied from `InputParameters` above, but it seems like a good time to fix this up.